react/react-training/src/screens/MyScreenComponent/Square.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Remember change the name of the file for `index.js` and this is a component of `Board`, check my comment above